Misc/Lobby/oa_tanabata.csv

```diff
@@ -131,19 +131,19 @@ result54#1,"""せーわ"""
 result55#1,"""機械仕掛けのさおリリーパ"""
 result56#1,"""クリきんと"""
 result57#1,"""mashiro@uyo"""
-result58#1,"""Cool _Nicos"""
+result58#1,"""cool _Nicos"""
```

Comment:
Undo the changes for this file
This is list of player names, aka, a highscore

Reply:
it's why we never bothered translating them